index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
     <link rel="stylesheet" href="./css/home-page.css" />
+    <link rel="icon" href="images/favicon.png"/>
```

Comment:
Kelias iki failo rekomenduotina, jog prasidėtų reliatyvia nuoroda, t.y. pvz `./folder/file` - esminis momentas - taškas.

Reply:
Supratau. Pataisiau, sucommitinau ir issiunciau.  Ar reikia is naujo pull request kurti?